src/main/java/cms/gov/madie/measure/config/AddOrganizationsChangeUnit.java

```diff
@@ -22,9 +22,11 @@ public AddOrganizationsChangeUnit(ObjectMapper objectMapper) {
 
   @Execution
   public void addOrganizations(OrganizationRepository organizationRepository) throws IOException {
-    InputStream inputStream = getClass().getResourceAsStream("/data/organizations.json");
```

Comment:
can we delete organization.json? or is it needed?

Reply:
I am not sure, if we need that. I can delete it